wordpress-mobile/WordPress-Android
# Issue/11769 feature announcement dialog list

Part of #11769
Based on #11770 which needs to be merged first.

Logic in `FeatureAnnouncementProvider` is pretty basic for now. 

I need to polish this a bit more for code review, but the design could be reviewed right now.

[![Image from Gyazo](https://i.gyazo.com/c73e5f150c9eb23550318981a2bbb4cc.png)](https://gyazo.com/c73e5f150c9eb23550318981a2bbb4cc)

[![Image from Gyazo](https://i.gyazo.com/eed5605883b019fd9ae8fd2b8b2b4685.png)](https://gyazo.com/eed5605883b019fd9ae8fd2b8b2b4685)

To test:
- Open Feature Announcement dialog from App Settings.
- Make sure the list looks ok in light/dark modes and in portrait/landscape orientations.
- Make sure `FIND OUT MORE` buttons opens wp blog.

PR submission checklist:
